docs/copied-from-beats/outputconfig.asciidoc

```diff
@@ -1569,7 +1569,9 @@ endif::[]
 
 ifdef::apm-server[]
 NOTE: This page refers to using a separate instance of APM Server with an existing Elasticsearch Service deployment.
```

Comment:
How about a link to ess here?  Maybe https://www.elastic.co/cloud/ ?

Reply:
Good idea! I think we're supposed to point ESS links here: https://www.elastic.co/cloud/elasticsearch-service
(Like on https://www.elastic.co/guide/en/apm/get-started/7.0/install-and-run.html)